index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ideefase PDF Generator</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.4.0/jspdf.umd.min.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/2.2.19/tailwind.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div class="container">
+    <form id="ideefaseForm" class="space-y-6" onsubmit="return validateForm();">
+    <!-- Header -->
+    <div class="header">
+        <header class="text-center">
+            <h1 class="title">Ideefase PDF Generator</h1>
+            <p class="subtitle">Genereer een overzichtelijk ideefase PDF</p>
+        </header>
+    </div>
+
+        <!-- Form Section -->
+    <div class="form-section">
+
+            <!-- Project Type -->
+            <div>
+                <label class="form-label">Project Type</label>
```

Comment:
Dit label hoort niet direct bij één van de inputs, dus ik denk dat je hier wellicht beter een `<h3>` of, zoals het 'echt' hoort, een `<fieldset>` (ipv div) met `<legend>` voor kunt gebruiken!